### PR TITLE
Add API to unregister window

### PR DESCRIPTION
**Pull request by hanbinhu**

### CODE REVIEW

**Review by ybc1991 — CHANGES_REQUESTED** [review]
I added a get_current_created_window_names function. I think it is better to use the global context to manage the unregister instead of the local map. Because the user may manually unregister the window

**Review by hanbinhu — COMMENTED** [review]
LGTM

**Comment by ybc1991** [comment]
I just added that function but didn't update the unregister functionality. You still need to update it and test it again

File: bluefog/torch/optimizers.py [hunk 1]
```diff
@@ -1261,6 +1259,7 @@ def DistributedWinPutOptimizer(optimizer, model,
                                      communication. This allows local model parameter updates
                                      per num_steps_per_communication before reducing them over
                                      distributed computation resources.
+        window_prefix: A string to identify the unique DistributedWinPutOptimizer.
```
Comment (ybc1991) [inline bluefog/torch/optimizers.py:1262]: A string to identify the unique DistributedWinPutOptimizer, which will be applied as the prefix for window name

**Review by ybc1991 — APPROVED** [review]

**Pull request merged**